### PR TITLE
Sprint #4

### PR DESCRIPTION
**Pull request by nadyavalin**
1. Task: [Sprint #4](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/eCommerce-Application/Sprints/Sprint%234.md)
2. Screenshots:
image
image
image
image
image

3. Deploy: [https://online-moon-store.netlify.app](https://online-moon-store.netlify.app/)
5. Done: 16.06.2024 / deadline 18.06.2024
6. Score: 45 / 60
- [x] Code Review Participation: Students actively participate in code reviews, providing constructive feedback and suggestions to their peers (10/10);
- [x] No Merges Without Approvals: Pull requests are not merged without approvals from other team members, ensuring the quality of code contributions (10/10);
- [x] Pull Request Descriptions: Pull requests include detailed descriptions, outlining the changes made and the purpose of the changes, facilitating better understanding and review (10/10);
- [x] Task Board Management: Students maintain and regularly update a task board (e.g., Trello, Jira), ensuring that tasks are moved according to their progress and completion (10/10);
- [x] Active Participation in Development Discussions: Students take an active role in discussing the development process, sharing ideas, and collaborating on solutions and improvements (5/5);
- [ ] Unit Test Coverage: The codebase has a minimum of 30% unit test coverage, ensuring the reliability and robustness of the implemented features (0/15).

### CODE REVIEW

File: src/pages/basket/basketHandler.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+export function increaseQuantityProduct(
+  btn: HTMLElement,
+  countDiv: HTMLElement,
+  totalPriceDiv: HTMLElement,
+  lineItemId: string,
+  lineItemIndex: number,
+) {
+  let quantity = Number(countDiv.textContent);
+  quantity += 1;
```
[the remark this answers is not in the document]
Comment (ifbfirst) [reply src/pages/basket/basketHandler.ts]: I want to refactor this function and merge two in one

File: src/pages/basket/basketHandler.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    updateCart(response.body.version, [
+      {
+        action: "changeLineItemQuantity",
+        lineItemId,
+        quantity,
+      },
+    ])?.then((response) => {
+      if (response.statusCode === 200) {
+        countDiv.textContent = `${quantity}`;
+        totalPriceDiv.textContent = `${PriceFormatter.formatCents(response.body.lineItems[+lineItemIndex].totalPrice.centAmount)}`;
```
[the remark this answers is not in the document]
Comment (ifbfirst) [reply src/pages/basket/basketHandler.ts]: It is the way to make string in number

[the remark this answers is not in the document]
Comment (arcanar7) [reply src/pages/basket/basketHandler.ts]: только он уже number `lineItemIndex: number,`

[the remark this answers is not in the document]
Comment (ifbfirst) [reply src/pages/basket/basketHandler.ts]: Ok, thanks, I will correct that

**Review by Shustovsky — APPROVED** [review]

**Review by arcanar7 — APPROVED** [review]

**Review by alex-mirankov — APPROVED** [review]

**Pull request merged**